src/gluonts/distribution/distribution.py

```diff
@@ -223,6 +230,12 @@ def cdf(self, x: Tensor) -> Tensor:
         """
         raise NotImplementedError()
 
+    def quantile(self, level: Tensor) -> Tensor:
+        r"""
+        Returns the quantile corresponding to the passed level
```

Comment:
It would be better to document `level` here as well as the return value (shape and so on)